src/frequenz/sdk/actor/power_distributing/request.py

```diff
@@ -17,8 +17,8 @@ class Request:
     power: Power
     """The requested power."""
 
-    batteries: abc.Set[int]
-    """The component ids of the batteries to be used for this request."""
+    component_ids: abc.Set[int]
+    """The component ids of the components to be used for this request."""
```

Comment:
You might want to mention this change in the release notes as `Request` is exposed

Reply:
I guess you consider `Request` and `PowerDistributingActor` as public types because they're in `actor/power_distributing`?

It is not supposed to be.  So I would instead rename it to `actor/_power_distributing` and mention that alone in the release notes.  That's how the `actor/_power_managing` actor is,  and this one is public because it is from a time when users were instantiating and using the actors directly.

Reply:
just amended the release notes commit: https://github.com/frequenz-floss/frequenz-sdk-python/commit/b9c8573a